pkg/manager/signals/signal_test.go

```diff
@@ -36,7 +36,7 @@ var _ = Describe("runtime signal", func() {
 			task := &Task{
 				ticker: time.NewTicker(time.Second * 2),
 			}
-			c := make(chan os.Signal)
+			c := make(chan os.Signal, 1)
```

Comment:
context on this one?

Reply:
`SA1017: the channel used with signal.Notify should be buffered`
https://staticcheck.io/docs/checks#SA1017

Reply:
ah, prevents missing signals, ok.